osu.Game/Online/Chat/ChannelManager.cs

```diff
@@ -256,8 +256,38 @@ public void PostCommand(string text, Channel target = null)
                     JoinChannel(channel);
                     break;
 
+                case "msg":
+                    if (string.IsNullOrWhiteSpace(content))
+                    {
+                        target.AddNewMessages(new ErrorMessage("Usage: /msg [user] [msg]"));
+                        break;
+                    }
+
+                    var args = content.Split(' ', 2);
```

Comment:
I feel like opening the channel without sending a message would be the right way to go about this instead of blocking the user's half-correct usage entirely.